frontend/src/utils/addressBlacklist.ts

```diff
@@ -0,0 +1,284 @@
+export const blacklistAddresses = [
```

Comment:
Nitpick: I would use UPPER_CASE for such constants.

Also, I am not sure if the `utils` folder is the right place. One approach would be to move such constant values to the place where they are used (separate file but same folder), but would not work if we need the check twice. Or we could only export a utility function `isAddressBlacklisted` which makes the complete call `blacklistAddresses.includes(getAddress(address))`. would also reduce the redundant `includes` calls (depending on if it is necessary to have the check twice)

Reply:
:heavy_check_mark: